spec/forms/claim_details_form_spec.rb

```diff
@@ -60,6 +61,17 @@
     end
   end
 
+  describe '#other_known_claimant_names' do
+    context 'when #other_known_claimant is false' do
+      it 'should not have a value' do
```

Comment:
just a wording thing - maybe add 'after validation' to this ?  it is not clear that validation is involved